packages/block-library/src/cover/edit.js

```diff
@@ -133,7 +140,15 @@ function CoverHeightInput( {
 		}
 	};
 
-	const inputValue = temporaryInput !== null ? temporaryInput : value;
+	const computedValue = useMemo( () => {
+		const inputValue = temporaryInput !== null ? temporaryInput : value;
+		const [ parsedQuantity ] = parseQuantityAndUnitFromRawValue(
+			inputValue
+		);
+
+		return [ parsedQuantity, unit ].join( '' );
+	}, [ temporaryInput, value ] );
```

Comment:
Do we need to add `unit` to the dependency array here? In this change, I couldn't find any issues that came up from not including it, but if we were to refactor to remove the `temporaryInput` internal state, then I was wondering if it could be an issue.